scripts/doc-linter.py

```
@@ -215,16 +247,21 @@ def handle_file_action(filepath):
         for file in files:
-                if file.endswith(SUPPORTED_EXTS) and "_build" not in file:
-                    if handle_file_action(file) is False:
-                        bool_error = False
+            if (
+                file.endswith(SUPPORTED_EXTS)
+                and "_build" not in file
+                and is_docs_path(file)
+            ):
+                if handle_file_action(file) is False:
+                    bool_error = False
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)